components/icons/BadgeCheckOutline.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2021 The Rode Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+
+const BadgeCheckOutline = () => {
```

Comment:
I didn't notice this before, but there's a package for these icons: [`@heroicons/react`](https://github.com/tailwindlabs/heroicons#react).

They don't have the title element, but apparently that's intentional: 

```
https://github.com/tailwindlabs/heroicons/issues/111
```

Do you know if there are other differences between the vendored versions in `components/icons` and the package?

Reply:
Oh wow I did not realize. I've been adding the titles to be able to distinguish between icons when testing but I can look into other ways of doing that. I'll make an issue to use this package instead of making icons ourselves.

Reply:
Made this [issue](https://github.com/rode/rode-ui/issues/149) but I forgot about Josh possibly replacing the icons sometime with custom ones so I likely won't prioritize this next week.